docs/openapi/api-internal-v1.yaml

```diff
@@ -0,0 +1,410 @@
+      tags:
+        - Write
+      operationId: addStatusChangeEvent
+      parameters:
+        - $ref: 'remote-refs.yaml#/components/parameters/uidAuthFleet'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: "#/components/schemas/PnStatusUpdateEvent"
+        required: true
+      responses:
+        '204':
+          description: OK
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: './schemas-pn-errors-v1.yaml#/components/schemas/Problem'
+
+components:
+  parameters:
+    
+    ###########################################################################################
+    ###                          PARAMETRI DI RICERCA DEI DOWNTIME                          ###
+    ###########################################################################################
+    
+    queryFromTime:
+      name: fromTime
+      in: query
+      required: true
+      description: data/ora di inizio dell'intervallo entro cui eseguire la ricerca
+      schema:
+        type: string
+        format: date-time
+    
+    queryToTime:
+      name: toTime
+      in: query
+      required: false
+      description: data/ora di fine dell'intervallo entro cui eseguire la ricerca
+      schema:
+        type: string
+        format: date-time
+    
+    queryFunctionality:
+      name: functionality
+      in: query
+      required: false
+      description: funzionalità di Piattaforma Notifiche di cui elencare i disservizi
+      schema:
+        type: array
+        items:
+          $ref: '#/components/schemas/PnFunctionality'
+    
+    queryPage:
+      name: page
+      in: query
+      required: false
+      description: Pagina di risultati a cui il client è interessato
+      schema:
+        type: string
+
+
+  schemas:
+    
+    ###########################################################################################
+    ###                DTO VISUALIZZAZIONE STATO DELLA PIATTAFORMA NOTIFICHE                ###
+    ###########################################################################################
+    PnStatusResponse:
+      title: Stato attuale di PN
+      description: >-
+        Elenco delle funzionalità della piattaforma ed elenco dei disservizi noti e attivi 
+        al momento della richiesta. I disservizi (_openIncidents_) segnalati sono al più uno 
+        per funzionalità; gli attributi _endDate_ e _legalFactId_ non saranno valorizzati.
+      type: object
+      required:
+        - functionalities
+        - openIncidents
+      properties:
+        functionalities:
+          title: elenco funzionalità di PN
+          description: Un array che comprende tutti i possibili valori dell'enum _PnFunctionality_
+          type: array
+          items: 
+            $ref: '#/components/schemas/PnFunctionality'
```

Comment:
Si da per scontato che quello che non appare in openIncidents stia funzionando?
Non è più semplice mettere una lista con OK e KO?

Reply:
Si da per scontato che quello che non appare in openIncidents stia funzionando? SI
Dal punto di vista dell'implementazione di back-end è più semplice mettere solo gli errori. 

Inoltre ritengo sia un informazione utile sapere "da quanto tempo è rotto" mentre avere la lista degli OK con "da quanto tempo è OK" sarebbe oneroso e poco utile. Questo implica comunque un asimmetria nelle API.